.openshift-ci/sensor_versions.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python3
+
+"""
+Returns the latest patches of the last 4 major versions
+"""
+
+import sys
+import re
+import subprocess
+
+def isReleaseTagGitCLI(version):
+    return bool(re.search(r"^\d+\.\d+\.\d+$", version))
+
+def filterGitCLITags(rawtags):
+    filteredtags = []
+    for t in rawtags:
+        if isReleaseTagGitCLI(t):
+            filteredtags.append(t)
+    return set(filteredtags)
```

Comment:
[nerd mode: on] A more idiomatic Python would be
```suggestion
    return set([t for t in rawtags if isReleaseTagGitCLI(t)])
```

Reply:
Very cool, I'll take it 👍